web/github.go

```diff
@@ -70,45 +132,159 @@ func (g *GitHub) Orgs(w http.ResponseWriter, r *http.Request) {
 
 	// initialized as empty array because otherwise json response will be null
 	// instead of []
-	orgs := []orgRespose{}
+	orgs := []orgsListItem{}
 
 	for _, installation := range installations {
-		// New transport for each installation
-		itr, err := ghinstallation.NewKeyFromFile(
-			http.DefaultTransport, g.AppID, int(installation.GetID()), g.PrivateKey)
-
-		if err != nil {
-			ctxlog.Get(r.Context()).Errorf(err, "failed to initialize the GitHub App installation client")
-			http.Error(w, http.StatusText(http.StatusInternalServerError), http.StatusInternalServerError)
-			return
-		}
-
-		// Use installation transport in a new client
-		client := github.NewClient(&http.Client{Transport: itr})
-
-		if installation.Account == nil || installation.Account.Login == nil {
-			ctxlog.Get(r.Context()).Errorf(nil, "failed to get GitHub installation organization name")
-			http.Error(w, http.StatusText(http.StatusInternalServerError), http.StatusInternalServerError)
-			return
-		}
-
 		// a GitHub App can also be installed in a User account
-		if *installation.Account.Type != "Organization" {
+		if installation.GetAccount().GetType() != "Organization" {
 			continue
 		}
 
-		org := *installation.Account.Login
-		mem, _, err := client.Organizations.GetOrgMembership(r.Context(), login, org)
+		admin, err := g.admin(r.Context(), installation, login)
```

Comment:
since `admin` returns a boolean, I wonder if it would be better `isAdmin`
```suggestion
		admin, err := g.isAdmin(r.Context(), installation, login)
```